packages/core/src/grid/docs/row-single-select.story.ts

```diff
@@ -15,13 +15,13 @@ export function rowSingleSelect() {
           <cds-grid-column type="action"></cds-grid-column>
           ${this.grid.columns.map(column => html`<cds-grid-column>${column.label}</cds-grid-column>`)}
           ${this.grid.rows.map(
-            row => html` <cds-grid-row .selected=${row.selected}>
+            row => html` <cds-grid-row .selected=${row.selected as boolean}>
               <cds-grid-cell>
                 <cds-radio>
                   <input
                     type="radio"
                     name="row"
-                    .checked=${row.selected}
+                    .checked=${row.selected as boolean}
```

Comment:
These cases I will work on cleaning up/improving but this is fine for the PR. Its due to the test data being optional/undefined while the component expects a value regardless if it coerces to a bool.